app/templates/search/_opportunity_data.html

```diff
@@ -0,0 +1,18 @@
+<br />
+<div class="dmspeak">
+  <h3 class="heading-small opportunity-data-header">Opportunity data</h3>
+  <p class='opportunity-data-description'>Download data about closed opportunities including award status. This will be updated weekly.<p>
+</div>
+{%
+with
+items = [
+    {
+        "title": "Download data",
+        "link": "https://assets.digitalmarketplace.service.gov.uk/digital-outcomes-and-specialists-2/communications/data/opportunity-data.csv",
+        "file_type": "CSV",
+        "download": "True"
+    }
+]
```

Comment:
It might be better to add analytics using the following example rather than adding a new function:

https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/639/files#diff-59a410eae02a55237a24954eef533e1fR75